package.json

```diff
@@ -33,7 +33,7 @@
         "eslint": "eslint \"./src/*.js\" \"./src/**/*.js\" --report-unused-disable-directives",
         "prettier": "prettier \"./src/*.{css,html,js}\" \"./src/**/*.{css,html,js}\" \"./.circleci/config.yml\" \"./package.json\"",
         "prettier:write": "npm run prettier -- --write",
-        "start": "http-server"
+        "start": "rollup --config"
```

Comment:
Are there updates that should happen to README.md for this?

Reply:
yep, good call. Updated.